### ISSUE TITLE
Metadata: Read GPS information from Apple XMP

### ISSUE THREAD

**Issue by graciousgrey** [photoprism/photoprism#2828]
Acceptance Criteria:
- [ ] Read GPS information from apple xmp
- [ ] Write unit test
- [ ] Update documentation

Follow up for #636

**Comment by gacolitti** [comment]
I'm considering migrating a large Apple Photos library to PhotoPrism. I want to retain GPS information. If I migrate now, GPS data will be lost and there is no easy way to add it in the future, correct? So it's best to wait for this issue to be closed first.

**Comment by gacolitti** [comment]
I followed the migrating from Apple Photos [guide](https://docs.photoprism.app/user-guide/use-cases/apple/), which seems to suggest GPS data is not currently read. But I see location info (including Lat and Long) in PhotoPrism after importing. Is this information read from somewhere other than Apple XMP?

**Comment by graciousgrey** [comment]
Yes, GPS data is read from your files EXIF data, see https://photoprism.app/kb/metadata & https://docs.photoprism.app/user-guide/library/metadata/

**Comment by lastzero** [comment]
Improved support for reading metadata from XMP sidecar files has been merged into `develop`, so it will ship with the next stable release and can be tested using our preview builds:

- #5563
- https://docs.photoprism.app/release-notes/#development-preview

Issue #2260 has been turned into an epic to track XMP-related enhancements and feature requests:

- #2260
